packages/app/ui/inputs/TextInput.tsx

```diff
@@ -0,0 +1,41 @@
+import { Input, InputProps, Field, Label } from "@headlessui/react";
+import { twMerge } from "tailwind-merge";
+
+import { BodyText, CaptionText } from "../text";
+
+interface TextInputProps extends InputProps {
+  className?: string;
+  error?: string;
+  id: string;
+  label?: string;
+}
+
+export const TextInput = ({
+  className,
+  error,
+  id,
+  label,
+  ...inputProps
+}: TextInputProps) => (
+  <Field className="flex flex-col gap-2">
+    {label && (
+      <Label htmlFor={id}>
+        <BodyText className="text-em-low">{label}</BodyText>
+      </Label>
+    )}
+    <Input
+      id={id}
+      className={twMerge(
+        [
+          "border placeholder:text-sm w-full min-h-10 rounded-3xl px-3 py-2",
+          "border-surface-75 placeholder:text-em-low outline-primary-500",
```

Comment:
why have 2 strings? 

At least let's split it per context, I would prefer something like this:

```suggestion
          "border border-surface-75",
          "placeholder:text-sm placeholder:text-em-low ",
          "outline-primary-500   w-full min-h-10 rounded-3xl px-3 py-2"
```

Reply:
I like the suggestion, I tried to group colors on one hand and the rest on the other, but I like the context idea more.

I'll move the `outline-primary-500" color to the border context because it's kinda related.